build-id-versioning/starter/main.go

```diff
@@ -0,0 +1,132 @@
+package main
+
+import (
+	"context"
+	"log"
+	"os"
+	"time"
+
+	"github.com/pborman/uuid"
+
+	"go.temporal.io/sdk/client"
+)
+
+func main() {
+	ctx := context.Background()
+
+	// Get task queue name from CLI arg
+	taskQueue := os.Args[1]
+	if taskQueue == "" {
+		log.Fatalln("Must provide task queue name as first and only argument")
+	}
+	log.Println("Using task queue", taskQueue)
+
+	// The client is a heavyweight object that should be created once per process.
+	c, err := client.Dial(client.Options{
+		HostPort: client.DefaultHostPort,
+	})
```

Comment:
```suggestion
	c, err := client.Dial(client.Options{})
```

(sorry, I know we probably have this default elsewhere and this was just pasted from there, don't have to fix)

Reply:
I kinda like that we're putting the HostPort in the samples, it teaches where this option can be provided and what the default is.

Reply:
But this doesn't show what the default is, just puts a constant reference. Are you saying we should put `"localhost:7233"` here and in future examples for clarity reasons? Doesn't matter much either way to me.

Reply:
I mean it's a "go to definition" (or 2 with the whole internal situation) away but in any case this is insignificant.